pkg/virt-handler/node-labeller/cpu_plugin.go

```diff
@@ -111,6 +111,7 @@ func (n *NodeLabeller) loadDomCapabilities() error {
 					n.hostCPUModel.requiredFeatures[feature.Name] = true
 				}
 			}
+			usableModels = append(usableModels, hostCpuModel.Name)
```

Comment:
nit: I think it would be clearer to move to below `hostCpuModel := mode.Model[0]`